openedx/features/course_experience/views/course_home_messages.py

```diff
@@ -177,7 +169,7 @@ def _register_course_home_messages(request, course_id, user_access, course_start
                 '{initial_tag}{goal_text}{closing_tag}'
             ).format(
                 initial_tag=HTML(
-                    '<div tabindex="0" aria-label="{aria_label_choice}" class="goal-option {col_sel} btn" '
+                    '<button tabindex="0" aria-label="{aria_label_choice}" class="goal-option {col_sel} btn" '
```

Comment:
Why are you explicitly setting the tabindex? Buttons are focusable by default.

Reply:
good catch, un-removed remnant from when it was a div.